__tests__/app.test.js

```diff
@@ -273,7 +273,121 @@ describe("app", () => {
         .send(newComment)
         .expect(400)
         .then(({ body }) => {
-          expect(body.msg).toBe("Bad request: invalid input");
+          expect(body.msg).toBe("Bad request");
+        });
+    });
+  });
+  describe.only("PATCH: /api/reviews/:review_id", () => {
```

Comment:
Need to get rid of .onlys before submitting so that we know all your tests are passing 🙂 
Could you remove and double check they're all passing before merging